nifi-nar-bundles/nifi-framework-bundle/nifi-framework/nifi-web/nifi-web-ui/src/main/webapp/js/nf/canvas/nf-queue-listing.js

```diff
@@ -204,32 +204,48 @@
             });
         }
 
-        // add an actions column when the user can access provenance
-        if (nfCommon.canAccessProvenance()) {
-            // function for formatting actions
-            var actionsFormatter = function () {
-                return '<div title="Provenance" class="pointer icon icon-provenance view-provenance"></div>';
-            };
+        // function for formatting actions column
+        var actionsFormatter = function (row,cell,value,columnDef,dataContext) {
+            var formatted = '';
+
+            var disabled = (dataContext.size > 0)?false:true;
+            formatted += '<div class="icon download-flowfile-content fa fa-download '+((disabled)?'disabled':'pointer')+'"'+
```

Comment:
I was surprised to see that there is no security check, but this is how it happens in the details view also, so I guess it's good.